components/Common/Tabs/index.tsx

```diff
@@ -9,33 +9,42 @@ type Tab = {
   label: string;
 };
 
-type TabsProps = {
+export type TabsProps = {
```

Comment:
No export of Component Types, if you need them, simply do `ComponentProps<typeof Component>`